code/controllers/CronTaskController.php

```diff
@@ -9,21 +10,11 @@ class CronTaskController extends Controller
 {
 
     /**
-     * If this controller is in quiet mode
+     * Setting this to true will make the controller print debug information
      *
      * @var bool
      */
-    protected $quiet = false;
```

Comment:
This becomes a major API change. Any reason you didn't just change this to `protected $quiet = true`?